src/components/Dropdown/Dropdown.tsx

```diff
@@ -80,8 +80,12 @@ export const Dropdown: FC<DropdownProps> = React.memo(({
     setVisible(false);
   };
 
-  const handleDropdownClick = () => {
+  const handleDropdownClick = async () => {
     setVisible((oldValue) => !oldValue);
+
+    if (updatePopper != null) {
```

Comment:
```suggestion
    if (updatePopper) {
```

Why not only this? 
Because the current implementation may fail if `updatePopper` is `undefined`

Reply:
it's not a strict equality, so it's also considering `updatePopper !== undefined`